README.md

```diff
@@ -1,4 +1,4 @@
-# Logbook
+## Logbook: HTTP request and response logging

```

Comment:
Ah, I had shortened it but because of the old change request couldn't tell if I'd gotten it down to a single line. We can just keep Logbook for now and think about it a bit later, wdyt?